src/Main.cpp

```diff
@@ -262,20 +266,20 @@ int main(int argc, char** argv) {
 
     static MessageDispatcherTask s_hostDispatchTask("tcp_dispatch", gc_taskNormalPriority, NULL,
                                                     MessageHandlerContainer::getHostMsgQueue());
-    static MessageSenderTask s_hostMessageSender("host_send", gc_taskNormalPriority, NULL,
-                                                 MessageHandlerContainer::getHostMsgQueue());
+    static MessageSenderTask<HiveMindHostSerializer> s_hostMessageSender(
+        "host_send", gc_taskNormalPriority, NULL, MessageHandlerContainer::getHostMsgQueue());
     static MessageDispatcherTask s_remoteDispatchTask("remote_dispatch", gc_taskNormalPriority,
                                                       NULL,
                                                       MessageHandlerContainer::getRemoteMsgQueue());
-    static MessageSenderTask s_remoteMessageSender("remote_send", gc_taskNormalPriority, NULL,
-                                                   MessageHandlerContainer::getRemoteMsgQueue());
-
-    static CommMonitoringTask s_hostMonitorTask("host_monitor", gc_taskNormalPriority,
-                                                s_hostDispatchTask, s_hostMessageSender,
-                                                BSPContainer::getHostCommInterface);
-    static CommMonitoringTask s_remoteMonitorTask("remote_monitor", gc_taskNormalPriority,
-                                                  s_remoteDispatchTask, s_remoteMessageSender,
-                                                  BSPContainer::getRemoteCommInterface);
+    static MessageSenderTask<HiveMindHostAccumulatorSerializer> s_remoteMessageSender(
+        "remote_send", gc_taskNormalPriority, NULL, MessageHandlerContainer::getRemoteMsgQueue());
```

Comment:
What should be not normal priority?

Reply:
Tasks of higher priorities like interloc. This "normal" priority is basically the default one that we use for the dispatcher and message senders